src/common/Configuration/Config.cpp

```diff
@@ -144,29 +144,17 @@ namespace
 
             // read line error
             if (!in.good() && !in.eof())
-            {
                 throw ConfigException(Acore::StringFormatFmt("> Config::LoadFile: Failure to read line number {} in file '{}'", lineNumber, file));
-            }
 
             // remove whitespace in line
             line = Acore::String::Trim(line, in.getloc());
 
             if (line.empty())
-            {
                 continue;
-            }
 
-            // comments
+            // comments and headers
             if (line[0] == '#' || line[0] == '[')
-            {
                 continue;
-            }
-
-            size_t found = line.find_first_of('#');
-            if (found != std::string::npos)
-            {
-                line = line.substr(0, found);
-            }
```

Comment:
The purpose of this was to remove comments after a set value, like

`confOption = value # Comment get removed`

But IMO we do not need to set comments after a set value, which we don't, so we can support having it in for example passwords